travvy/src/components/Onboarding.vue

```diff
@@ -16,14 +16,14 @@
       </u1>
     </p>
     <br>
-    <label><strong>Please verify your email here (required): </strong></label>
+    <label><strong>Please verify your email here: * </strong></label>
     <input type="text" class="info" v-model="email" placeholder="Input Email here" required>
     <br>
     <br>
     <br>
 
     <div class= "dropwdown">
-    <label><strong>Please select the gender you identify as (required): </strong></label>
+    <label><strong>Please select the gender you identify as: * </strong></label>
```

Comment:
Glad this change was made based on the feedback we received the last sprint, thanks Rushan!